numpy/core/code_generators/ufunc_docstrings.py

```diff
@@ -4232,3 +4232,37 @@ def add_newdoc(place, name, doc):
     array([ 0, 20, 20, 60, 20, 20])
 
     """)
+
+add_newdoc('numpy.core.umath', 'bit_count',
+    """
+    Computes the number of 1-bits in ``x``.
+    Analogous to the builtin `int.bit_count` or ``popcount`` in C++.
```

Comment:
Should we mention that the sign is ignored (i.e. for signed integers the abs-value is used)?

Reply:
I shall add it 👍

Reply:
Fixed in 6d9acdd